eth_crowdfund_api/src/views/CampaignView.py

```diff
@@ -1,61 +1,62 @@
 from flask import request, json, Response, Blueprint, jsonify
 from ..models.Campaign import Campaign, CampaignSchema
-
+from marshmallow import ValidationError
+from sqlalchemy import exc
```

Comment:
Exception tools that come marshmallow and sqlalchemy

Reply:
very convenient